tests/flow.html

```diff
@@ -74,28 +71,15 @@ <h3>
       }
       // If we are loading from a CDN, build the URLs.
       if (typeof cdnURLPath !== 'undefined') {
+        // We only support version 4 or newer.
         const version = ver.split('@')[1];
-        if (parseFloat(version) < 4) {
-          isVersionFourOrNewer = false;
-          vexURL = `${cdnURLPath}vexflow@${version}/releases/vexflow-debug.js`;
-          testsURL = `${cdnURLPath}vexflow@${version}/releases/vexflow-tests.js`;
-        } else {
-          // VexFlow 4 moved the build output from releases/ to build/.
-          vexURL = `${cdnURLPath}vexflow@${version}/build/vexflow-debug.js`;
-          vexPlusTestsURL = `${cdnURLPath}vexflow@${version}/build/vexflow-debug-with-tests.js`;
-        }
```

Comment:
Remove some legacy code from flow.html. From now on we only support versions >= 4.0.